### ISSUE TITLE
Support icons on project templates

### ISSUE THREAD

**Issue by ciyer** [SwissDataScienceCenter/renku-ui#1467]
## Motivation

It is easier to identify a project template that has an icon.

## Mock-ups / Design

The proposal is to add images to the information that templates provide and display the images. If no image is available, a stock image should be used. Next to the create button, the long description for the template should be shown.

![image](https://user-images.githubusercontent.com/1196411/130751737-98180d9d-3e96-455a-9982-e4de1f2b375a.png)

## Progress

- [x] API Design (sprint-??)
- [x] UI Implementation

**Comment by lorenzo-cavazzi** [comment]
I think the easiest and safest solution would be to get the encoded image from the `templates.read_manifest` API.
The image could be referenced in the manifest.yaml as a local file in the templates repository, and the core service could read and encode it in the response body, maybe with a reasonable file size limit (or an encoder that reduces its footprint when needed).
The API JSON response would then include a new `icon` field.
```
{
    name,
    folder,
    description,
    variables,
    icon: <encoded image>
}
```

The alternative would be getting the URLs where we can download the images.
This simplifies the backend implementation, but it introduces a few additional issues since we don't have any control over the target (E.G. the image may be deleted and not available anymore at the target URL, or inaccessible from some networks, or we may easily hit API limits as it happened with GitHub when we used to fetch the files for the projects created in the UI)

**Comment by lorenzo-cavazzi** [comment]
closed in #1530
